Modifications

```diff
@@ -1 +0,0 @@
-
```

Comment:
What is the point of this file?

Reply:
It's where we will keep our modified maps and vehicles

Reply:
OK, it used to be a file and now it's a folder.